app/src/main/java/com/nitsilchar/hp/passwordStorage/activity/LoginActivity.java

```diff
@@ -30,7 +30,7 @@
     String str_Email;
     String str_Password, str_getEmail, str_getPass;
     EditText edt_Email, edt_Password;
-    Button login,register;
+    Button login,register, toreg;

```

Comment:
`toreg` is kind of difficult to intuitively understand. What does this button do. `login`, `register` make sense because the developer knows what they do. Any way to name `toreg` something a little more clear?

Reply:
I used 'newuser'. That seems understandable?